src/main/java/uk/gov/companieshouse/model/Constants.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.model;
+
+public final class Constants {
+
+    private Constants() {}
```

Comment:
do you need this? A default constructor is automatically available if not provided

Reply:
A default constructor in Java is public. This one is private, forbidding object creation of this class.

Reply:
Coolio